### DISCUSSION TITLE
Windows

### DISCUSSION THREAD

**Discussion by louisvangeldrop** [facebook/hermes#1545] [Static Hermes]
Is there already a working install for Windows?

**Comment by tmikov** [comment]
What do you mean? There are Windows binaries, for example here https://github.com/facebook/hermes/releases/tag/v0.13.0, or with RN.

**Reply by louisvangeldrop** [reply]
I mean a binary with static hermes.

**Reply by tmikov** [reply]
Ah, sorry. No, there is no pre-compiled binary of Static Hermes for Windows (but then again, there are no precompiled binaries for any platform yet)

**Reply by louisvangeldrop** [reply]
Actually I was referring to the "BuildingandRunning.md" installation instructions for building static hermes. 
I have looked at #1247, where you mentioned to fix the installation for Windows before the middle of the year.
